assets/i18n/qqq.json

```diff
@@ -512,5 +512,6 @@
 	"I18N_TOPNAV_SIGN_IN_WITH_GOOGLE": "Text displayed inside the Sign In dropdown. - When clicked, the user is signed in with Google Oauth - The text in the button needs to be less than 25 characters long",
 	"I18N_TOPNAV_TEACH_WITH_OPPIA": "Text displayed inside drop-down menu - The list is shown after the user hovers over the I18N_TOPNAV_ABOUT button. When the option is clicked, the teach with oppia page is loaded which contains information specific for teachers who would like to create explorations.",
 	"I18N_TOTAL_SUBSCRIBERS_TEXT": "Text displayed under the subscribers tab in creator dashboard. Tells the creator the total number of subscribers he/she has.",
+	"I18N_TUTOR_CARD_TOOLTIP": "Please don't enter any personal information here. Your answer will be saved and can be read by the creator.",
```

Comment:
qqq.json is for documentation, so you have to write what exactly `I18N_TUTOR_CARD_TOOLTIP` does.

Reply:
Thanks! I'll make changes accordingly.